### DISCUSSION TITLE
first Login/Register not possible - Error in the Log

### DISCUSSION THREAD

**Discussion by Movie62** [thedevs-network/kutt#674] [Q&A]
Hello, I have installed kutt.it on a server with Docker using a tutorial. The URL shorter works even without login. Unfortunately I can't login/register with the ADMIN email address I have. I hope you can help me!

Thats the Error i found with porainer in the Log: 
npm ERR! Failed at the kutt@2.7.4 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-01-31T20_45_16_920Z-debug.log
wait-for-it.sh: waiting 15 seconds for postgres:5432
wait-for-it.sh: postgres:5432 is available after 0 seconds
kutt@2.7.4 start /usr/src/app
npm run migrate && cross-env NODE_ENV=production node production-server/server.js
kutt@2.7.4 migrate /usr/src/app
knex migrate:latest --env production
Requiring external module ts-node/register
Using environment: production
Already up to date
Ready on http://localhost:3000/

**Comment by bpatrik** [comment]
I know its an old discussion, but I run into the same issue. The solution was to put the app behind ssl.
